### ISSUE TITLE
Windows fallback location?

### ISSUE THREAD

**Issue by Arcanemagus** [atom/dowsing-rod#3]
Thoughts on having a fallback `C:\Python27\python.exe` location inserted for Windows if the `python` on the `%PATH%` is Python 3? This is where `python` v2 will be installed in virtually every case (other than running inside a `venv`).

**Comment by smashwilson** [comment]
Yeah, that's a good idea. In general it'd be nice to have platform-specific sets of python 2 installation locations that we could probe in addition to the `${PATH}`. On MacOS, `brew install python@2` doesn't automatically add a binary to your path, for example, so we could save people an installation step and minimize our interference with their system environment.
